falcon/request_helpers.py

```diff
@@ -157,6 +157,24 @@ def write(self, data):
 
         raise IOError('Stream is not writeable')
 
+    def exhaust(self, chunk_size=64 * 1024):
+        """Exhaust the stream. This consumes all the data left until the
```

Comment:
Nitpick/Style: The first line should not wrap (simply put the second sentence on its own line, i.e.:

```py
"""Exhaust the stream. 

Consume all of the remaining data, up to the content size limit.

...

"""
```

Reply:
Done